src/App.css

```diff
@@ -1,4 +1,4 @@
-.App {
+/* .App {
   text-align: center;
   display: flex;
-}
+} */
```

Comment:
[OPTIONAL] Please remove unwanted commented code from your codebase it makes your codebase messy so kindly remove it if you want to check your previous code then check your GitHub commit history  :slightly_smiling_face:

Reply:
Thank you  so much. I will do that